tests/integration/maml_miniimagenet_test.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+
+import unittest
+import random
+
+import numpy as np
+import torch as th
+from torch import nn
+from torch import optim
+
+import learn2learn as l2l
+
+
+def accuracy(predictions, targets):
+    predictions = predictions.argmax(dim=1).view(targets.shape)
+    return (predictions == targets).sum().float() / targets.size(0)
+
+
+def fast_adapt(adaptation_data, evaluation_data, learner, loss, adaptation_steps, device):
+    for step in range(adaptation_steps):
+        data = [d for d in adaptation_data]
+        X = th.cat([d[0].unsqueeze(0) for d in data], dim=0).to(device)
+        y = th.cat([th.tensor(d[1]).view(-1) for d in data], dim=0).to(device)
+        train_error = loss(learner(X), y)
+        train_error /= len(adaptation_data)
+        learner.adapt(train_error)
+    data = [d for d in evaluation_data]
+    X = th.cat([d[0].unsqueeze(0) for d in data], dim=0).to(device)
+    y = th.cat([th.tensor(d[1]).view(-1) for d in data], dim=0).to(device)
+    predictions = learner(X)
+    valid_error = loss(predictions, y)
+    valid_error /= len(evaluation_data)
+    valid_accuracy = accuracy(predictions, y)
+    return valid_error, valid_accuracy
+
+
+def main(
+        ways=5,
+        shots=5,
+        meta_lr=0.003,
+        fast_lr=0.5,
+        meta_batch_size=32,
+        adaptation_steps=1,
+        num_iterations=60000,
+        cuda=False,
+        seed=42,
+):
+    random.seed(seed)
+    np.random.seed(seed)
+    th.manual_seed(seed)
+    device = th.device('cpu')
+    if cuda and th.cuda.device_count():
+        th.cuda.manual_seed(seed)
+        device = th.device('cuda')
+
+    # Create Datasets
+    train_dataset = l2l.vision.datasets.MiniImagenet(root='./data', mode='train')
+    valid_dataset = l2l.vision.datasets.MiniImagenet(root='./data', mode='validation')
+    test_dataset = l2l.vision.datasets.MiniImagenet(root='./data', mode='test')
+    train_dataset = l2l.data.MetaDataset(train_dataset)
```

Comment:
```python
    train_dataset = l2l.data.MetaDataset(train_dataset, labels_to_indices=train_dataset.data['class_dict'])
```

Assuming everything works in the new PR that I had created, the suggested change should save us a lot of time.